### PR TITLE
Add Formula for PHP 7 uuid

### PR DESCRIPTION
**Pull request by romainneutron**

### CODE REVIEW

File: Formula/php70-uuid.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
+require File.expand_path("../../Abstract/abstract-php-extension", __FILE__)
+
+class Php70Uuid < AbstractPhp70Extension
+  init
+  desc "UUID extension"
+  homepage "https://pecl.php.net/package/uuid"
+  url "https://pecl.php.net/get/uuid-1.0.4.tgz"
+  sha256 "63079b6a62a9d43691ecbcd4eb52e5e5fe17b5a3d0f8e46e3c17ff265c06a11f"
+
+  stable do
```
Comment (ablyler) [inline Formula/php70-uuid.rb]: Can you add a comment on what this patch is for?

File: Formula/php70-uuid.rb [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    patch do
+      url "https://gist.githubusercontent.com/romainneutron/fe068c297413aee565d5/raw/28d6ba0b6e902e82e71bb9a1ed768c836a8161e4/php-uuid-1.0.4"
+      sha256 "5f0664d5c4f55d4f6c037dab9f198e697afa3f9266854ed3945d7697fdb692b2"
+    end
+  end
+
+  head do
```
Comment (ablyler) [inline Formula/php70-uuid.rb]: Can you add a comment on what this patch is for?

Comment (romainneutron) [inline Formula/php70-uuid.rb]: fixed

**Comment by romainneutron** [comment]
PR updated, tests should be okay now, let's see :)

**Comment by ablyler** [comment]
@romainneutron thanks! can you run `brew audit homebrew/php/php70-uuid --strict` and fix the issues that it outputs as well?

**Comment by romainneutron** [comment]
woops, fixed @ablyler , waiting for tests :)

**Pull request closed without merging**